tests/FunctionalClientTest.php

```diff
@@ -296,7 +294,7 @@ public function testExecStreamStderrCustomEventWhileRunning($container)
         $stream->on('error', $this->expectCallableNever());
         $stream->on('end', $this->expectCallableOnce());
 
-        $output = Block\await(Stream\buffer($stream), Loop::get());
+        $output = \React\Async\await(Stream, Loop::get());
```

Comment:
Was getting rid of `\buffer($stream)` intentional? There's also a `Loop::get()` leftover.